1-CONTRIBUTION-GUIDE/README.md

```diff
@@ -17,7 +17,7 @@ To deploy a sample using the Azure Portal, click the **Deploy to Azure** button
 
```

Comment:
I considered adding a main.bicep file to 100-blank-template, but that would be confusing because it would be empty and I'd need to show azdeploy.json as compiled from bicep.  But I have to have 100-blank-template be the example of where they start from, since we should encourage using bicep.  Do we need a separate example for a simple bicep sample?

Reply:
Not sure I follow - why not just have a blank bicep file like we do for json? I'm not sure it helps much since we probably have an empty file, but if it serves some purpose to add something we can do that too...

Reply:
If we have a blank bicep file, then the azdeploy.json will would theoretically need to match it (i.e. it would show the template hash/etc).  Would it be confusing if I just add a blank main.bicep but leave azdeploy.json as is?

Reply:
I went ahead and added blank main.bicep